index.html

```diff
@@ -1,48 +1,297 @@
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                <div class="item7"><button class="see-project">See project</button></div>
+            
+            
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works6" class="container8">
+
+
+                <div class="item8"> <img alt="work6" src="img/work6.jpg"> </div>
+
+                    
+                                
+                <div class="item8"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                    <div class=" item8 btn-group">
+                        <button class="languages"> Ruby on rails</button>
+                        <button class="languages"> css</button>
+                        <button class="languages"> JavaScript</button>
+                        <button class="languages"> html</button>
+                    </div>
+
+                    
+                    <div class="item8"><button class="see-project">See project</button></div>
+                
+                
+
+
+
+            </section>
+        </div>
+    </div>
+        
+    
+        <section id="aboutme" class="container9">
+
+                <div class="about item9"><h1>About me</h1> </div>
+
+                <div class="description item9"><p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have
+                    a project you need coded, don’t hestiate to contact me. </p>
+                </div>
+
+                <div class="item9">
+                    <button class="resume">My Resume</button>
+                </div>
+
+                <section class="item9 coding-languages">
+
+                    <div><img class="lang-icon" alt="lang icon" src="img/icon-languages.png"></div>
+
+                    <div><h2 class="res">Languages</h2></div>
 
+                    <div class="btn-group1">
+            
+                        <button class="languages1"> JavaScript</button>
+                        <button class="languages1"> Ruby</button>
+                        <button class="languages1"> Html</button>
+                        <button class="languages1"> CSS</button>
+                    </div>
+
+                </section>
+
+                <section class="item9 frameworks">
+                    <div><img class="lang-icon " alt="lang icon" src="img/icon-frameworks.png"></div>
+
+                    <div><h2 class="res">frameworks</h2></div>
+
+                    <div class="btn-group2">
+                        <button class="languages2"> Bootstrap</button>
+                        <button class="languages2"> Ruby on rails</button>
+                        <button class="languages2"> RSpec</button>
+                        <button class="languages2"> Capybara</button>
+                        <button class="languages2"> Selenium</button>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li, let us wrap the technologies with the `ul` and `li`